src/audio/eq_iir/iir_generic.c

```diff
@@ -44,11 +44,44 @@
 
 /* 32 bit data, 32 bit coefficients and 64 bit state variables */
 
-int32_t iir_df2t(struct iir_state_df2t *iir, int32_t x)
+int32_t iir_process_biquad(int32_t in, int32_t *coef, int64_t *delay)
```

Comment:
I am wondering if maybe this should be inline to negate the effects of it being called so often